File: BaseCureAPI/BaseCureAPI/DB/Models/AuthToken.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace BaseCureAPI.DB.Models
{
    public partial class AuthToken
    {
        public int AuthTokenId { get; set; }
        public string? Vrijednost { get; set; }
        public DateTime? VrijemeEvidentiranja { get; set; }
        public string? IpAdresa { get; set; }
        public string? Code2f { get; set; }
        public string? Is2fOtkljucan { get; set; }
        public int? KorisnikId { get; set; }
        
        public virtual Korisnici? Korisnik { get; set; }
    }
}

File: AngularApp/src/app/endpoints/authToken.ts
import { Korisnik } from './korisnik';

export interface AuthToken {
  authTokenID: number;
  vrijednost: string;
  korisnik?: Korisnik;
  vrijemeEvidentiranja: Date;
  ipAdresa?: string;
  code2F: string;
  is2FOtkljucan: boolean;
}
